### DISCUSSION TITLE
import error in next js

### DISCUSSION THREAD

**Discussion by ali9821** [phantom/docs#85] [Help & Support] [answered]
### Summary

I created the connector with web3-react and getting this error when try to import  : 
SyntaxError: Cannot use import statement outside a module

### Steps to Reproduce

import { MetaMask } from "@web3-react/metamask";
import { Phantom } from "web3-react-phantom";
import type { CoinbaseWallet } from "@web3-react/coinbase-wallet";
import { Web3ReactHooks, Web3ReactProvider } from "@web3-react/core";
import { hooks as metaMaskHooks, metaMask } from "./connectors/metaMask";
import { hooks as coinBaseHooks, coinbaseWallet } from "./connectors/coinBase";
import { hooks as phantomHooks, phantom } from "./connectors/phantom";

const connectors: [MetaMask | CoinbaseWallet | Phantom, Web3ReactHooks][] = [
  [metaMask, metaMaskHooks],
  [coinbaseWallet, coinBaseHooks],
  [phantom,phantomHooks]
];

export default function WalletsProvider({
  children,
}: {
  children: JSX.Element | JSX.Element[];
}) {
  return (
    <Web3ReactProvider connectors={connectors}>{children}</Web3ReactProvider>
  );
}

### Phantom Version

1.0.4

### Is there an existing discussion for this?

- [ ] I have searched the existing discussions

**Comment by bfriel** [comment, marked as answer]
Hi @ali9821, this looks like an import issue between your app and a third-party package. Can you please try the solutions mentioned in [this post](https://stackoverflow.com/a/69899085/6633895)?

**Reply by imran-khan362** [reply]
please issue me file
